contracts/staking/fuse/FuseStaking.sol

```diff
@@ -246,16 +246,21 @@ contract FuseStaking is
 		uint256 totalAmountOfFuseForFuseAcceptingFaucets = _getAmountOfFuseForAllFaucets();
 
 		uint256 earnings = _balance() - debtToStakers - debtToDAO;
+
+		uint256 keeperPartInFuse = earnings
+			- (earnings * (RATIO_BASE - keeperRatio)) / RATIO_BASE;
```

Comment:
why not simply keeperPart = earnings*keeperRatio/RATIO_BASE
also every where you use this calculation pattern

Reply:
done